torch/csrc/jit/python/init.cpp

```diff
@@ -719,6 +719,19 @@ void initJITBindings(PyObject* module) {
                 preserved_methods,
                 methods_to_optimize);
           })
+      .def(
+          "_clone_module_with_class",
```

Comment:
Can we make this API name scarier, to something like "_hack_do_not_use_clone_module_with_class" to avoid accidental misuse?